src/js/components/organisms/hero-homepage/hero-homepage.scss

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+$headerHeight: 60px;
```

Comment:
Used only once, no need to define the variable then.

Reply:
I'll add a comment below instead 👍